file_converter/Data.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace file_converter
+{
+    public class Data
```

Comment:
You can add a custom 2 arguments constructor, to simplify the creation of `Data` objects. You can further simplify it by creating a single argument constructor that receives `content` and calculates field names based on the passed content